disco_aws_automation/resource_helper.py

```diff
@@ -87,8 +84,7 @@ def throttled_call(fun, *args, **kwargs):
     (up to MAX_POLL_INTERVAL seconds).
     """
     max_time = 5 * 60
```

Comment:
While we are working on this, can we change this to be consistent with `MAX_POLL_INTERVAL`? Maybe use `MAX_POLL_INTERVAL` here instead of a new variable and change `MAX_POLL_INTERVAL` to be 5mins everywhere